docs/Architecture.md

```
@@ -0,0 +1,462 @@
+type RelayActionHandler struct {
+    name        string              // "game-a/buy_book"
+    description string
+    schema      map[string]interface{}
+    gameID      string              // "game-a"
+    client      *IntegrationClient
+}
+```
+
+#### Action Registration Flow:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)